CSD412webProject/CSD412webProject/Models/User.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Microsoft.AspNetCore.SignalR;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace CSD412webProject.Models
+{
+    public class User
+    {
+
+        public int Id { get; set; }
```

Comment:
Not sure that we need a setter for an ID ( I would prefer to assign it while initialization (creation of the object), than leaving it blank at the beginning , and then set it (someday in future)